### PR TITLE
Export 2021/09/10

### PR DESCRIPTION
**Pull request by nishant2718**
# Overview
This PR is for v38 strings, associated with this [JIRA ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-3101).

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -1540,6 +1552,10 @@
         <source>Blocked</source>
         <note>Title on tracking protection menu for blocked items.</note>
       </trans-unit>
+      <trans-unit id="Menu.TrackingProtection.Details.Verifier" xml:space="preserve">
+        <source>Verified by %@</source>
+        <note>String to let users know the site verifier</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:1557]: Please always explain what the placeholder will replaced by

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -1754,12 +1770,12 @@
         <note>Show Firefox Browser Privacy Policy page from the Privacy section in the settings. See https://www.mozilla.org/privacy/firefox/</note>
       </trans-unit>
       <trans-unit id="ProtectionStatusSheet.InsecureConnection" xml:space="preserve">
-        <source>Insecure Connection</source>
+        <source>Connection is not secure</source>
         <note>value for label to indicate user is on an unencrypted website</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:1774]: String ID should change here because this is a new string.

File: templates/firefox-ios.xliff [hunk 3]
```diff
@@ -1754,12 +1770,12 @@
...
       </trans-unit>
       <trans-unit id="ProtectionStatusSheet.SecureConnection" xml:space="preserve">
-        <source>Secure Connection</source>
-        <note>value for label to indicate user is on a secure ssl connection</note>
+        <source>Connection is secure</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:1777]: String ID needs to be updated here too

**Review by Delphine — CHANGES_REQUESTED** [review]

File: templates/firefox-ios.xliff [hunk 4]
```diff
@@ -254,11 +254,15 @@
         <source>Saved Logins</source>
         <note>Settings item for clearing passwords and login data</note>
       </trans-unit>
+      <trans-unit id="Spotlight Index" xml:space="preserve">
+        <source>Spotlight Index</source>
+        <note>Settings item for deleting the iOS search index of browsed webpages</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:259]: Please give some short explanation on what "Spotlight" on iOS is, and if that should be translated or not

**Review by Delphine — CHANGES_REQUESTED** [review]

File: templates/firefox-ios.xliff (hunk 2 above)
Comment (nishant2718) [reply templates/firefox-ios.xliff:1782]: Instead of updating the string ID, I've made new strings.

File: templates/firefox-ios.xliff (hunk 3 above)
Comment (nishant2718) [reply templates/firefox-ios.xliff]: Instead of updating the string ID, I've made new strings.

**Review by Delphine — APPROVED** [review]

**Pull request merged**